frontend/public/components/factory/table.tsx

```diff
@@ -235,6 +223,8 @@ export type RowFunctionArgs = {obj: object, index: number, columns: [], isScroll
 export type RowFunction = (RowFunctionArgs) => JSX.Element;
```

Comment:
This should be `export type RowFunction = (args: RowFunctionArgs) => JSX.Element;`.

Reply:
updated 👍